main.go

```diff
@@ -60,6 +62,8 @@ func main() {
 	flagSet.Var(&whitelistDomains, "whitelist-domain", "allowed domains for redirection after authentication. Prefix domain with a . to allow subdomains (eg .example.com)")
 	flagSet.String("keycloak-group", "", "restrict login to members of this group.")
 	flagSet.String("azure-tenant", "common", "go to a tenant-specific or common (tenant-independent) endpoint.")
+	flagSet.Var(&azureOidcGroups, "azure-permitted-groups", "restrict logins to members of these azure groups")
```

Comment:
I wonder if we could make these flags more generic, so they aren't provider specific?

Reply:
But this particular flag is Azure specific, isn't it? It's similar to the `keycloak-group` flag a couple of lines above.